component/main.jsonnet

```diff
@@ -0,0 +1,16 @@
+local com = import 'lib/commodore.libjsonnet';
+local kap = import 'lib/kapitan.libjsonnet';
+local kube = import 'lib/kube.libjsonnet';
+local prom = import 'lib/prom.libsonnet';
+local rl = import 'lib/resource-locker.libjsonnet';
+
+
+local inv = kap.inventory();
+local params = inv.parameters.cortex_tenant_ns_label;
+
+local rules = import 'rules.jsonnet';
+local capacity = import 'capacity.libsonnet';
+
+{
+  cortex_tenant_ns_label: import 'cortex-tenant-ns-label.jsonnet',
```

Comment:
For this component, it would be fine to move the contents of `cortex-tenant-ns-label.jsonnet` directly into `main.jsonnet`.